e2e/tests/api_tests/defaultJudgments/api_dj_2v1_test.js

```diff
@@ -23,7 +23,7 @@ Scenario('Default Judgment claim', async ({I, api}) => {
   await api.defaultJudgment(config.applicantSolicitorUser);
 });
 
-Scenario('Verify Direction order(summaryJudgmentDirectionsTask) Judge task', async ({I, api, WA}) => {
+Scenario.skip('Verify Direction order(summaryJudgmentDirectionsTask) Judge task', async ({I, api, WA}) => {
```

Comment:
Why is this test skipped altogether?

Reply:
Leftover on my part from local testing - removed the skip.